hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import typing as t
+from datetime import datetime
+from dataclasses import asdict, dataclass
+import uuid
+from boto3.dynamodb.conditions import Key
+from mypy_boto3_dynamodb.service_resource import Table
+
+from threatexchange.content_type.content_base import ContentType
+from threatexchange.signal_type.signal_base import SignalType
+
+from hmalib.common.models.models_base import DynamoDBItem
+
+"""
+# DynamoDB Table HMABanks
+
+## Models
+Stores banks and bank-members & bank-member signals.
+
+BankMemberSignals are signals extracted from the content in the bank member
+object. However, <signal_id> is unique for a signal_type, signal_value tuple.
+This is enforced by the 4th type of entry in the default table index.
+
+## Indices
+
+1. Default Table Index
+PK                        SK                             Item
```

Comment:
nit: table is hard to read. recommendation: make `Item` the first column and add a `----------` line between column headers and values.